qcodes/instrument_drivers/stanford_research/SR86x.py

```diff
@@ -969,18 +964,24 @@ def __init__(self, name, address, max_frequency, reset=False, **kwargs):
         data_channels.lock()
         self.add_submodule("data_channels", data_channels)
 
-        # Interface
-        self.add_function('reset', call_cmd='*RST')
-
-        self.add_function('disable_front_panel', call_cmd='OVRM 0')
-        self.add_function('enable_front_panel', call_cmd='OVRM 1')
```

Comment:
no bound methods for these ones? :(

Reply:
Bro, read the commit message :)

The underlying API does not work. I even have an email from SRS confirming this.

Reply:
oh, thanks, sorry )